eng/tools/versioning/getPackagePath.js

```diff
@@ -0,0 +1,34 @@
+let argv = require("yargs")
+  .options({
+    "package-name": {
+      type: "string",
+      describe:
+        "name of the artifact to be set (e.g. azure-keyvault-secrets), will be translated to @azure/(package) format",
+      demandOption: true
+    },
+    "repo-root": {
+      type: "string",
+      default: "../../../",
+      describe: "root of the repository (e.g. ../../../)",
+      demandOption: true
+    }
+  })
+  .help().argv;
+
+const versionUtils = require("./VersionUtils");
```

Comment:
This tool doesn't relate to versioning but it does require `VersionUtils` to read and parse a JSON file. Perhaps `getRushSpec` and the things it calls on should be refactored somewhere common?

Reply:
yeah may be we can track that in a separate issue? https://github.com/Azure/azure-sdk-for-js/issues/7472